metadata.json

```diff
@@ -31,21 +31,21 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.5.8 < 7.0.0"
+      "version_requirement": ">= 6.1.0 < 8.0.0"
     }
   ],
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.1 < 7.0.0"
+      "version_requirement": ">= 7.0.0 < 8.0.0"
```

Comment:
All the changes from these PR seems to have been in #317 except this lower bound.